pages/common/virt-sysprep.md

```diff
@@ -0,0 +1,20 @@
+# virt-sysprep
+
+> Reset, unconfigure, or customize a virtual machine image.
+> More information: <https://manned.org/virt-sysprep>.
+
+- List all supported operations (enabled operations are indicated with asterisks):
+
+`virt-sysprep --list-operations`
+
+- Run all enabled operations but don't apply the changes:
+
+`virt-sysprep --domain {{vm_name}} --dry-run`
+
+- Run only the specified operations:
+
+`virt-sysprep --domain {{vm_name}} --operations {{operation1,operation2,...}}`
+
+- Generate a new `/etc/machine-id` and enable customizations to be able to change the host_name to avoid network conflicts:
```

Comment:
Just one last change:
```suggestion
- Generate a new `/etc/machine-id` and enable customizations to be able to change the host name to avoid network conflicts:
```
In case you want to keep `host_name`, use ``` `host_name` ```.